### PR TITLE
Response to Reviewers

### PR DESCRIPTION
**Pull request by TanyaS08**

### CODE REVIEW

File: manuscript.md [hunk 1]
```diff
@@ -406,9 +406,17 @@ be no interaction between two species, it will be correct in the majority of
 cases because the majority of potential interactions in a network typically
 do not exist.  Therefore this "empty-matrix" model would always have an
 _accuracy_ of $1-C$, where $C$ is the observed connectance, which would almost
-always be greater than 50%. In response to the general problem of biases in
+always be greater than 50%. Understanding model performance within 
+sensitivity-specificity space may be more informative, where sensitivity 
+evaluates how good the model is at predicting true interactions (TPs) and 
+specificity refers to the prediction of true non-interactions (TNs).
```
Comment (TanyaS08) [inline manuscript.md:412]: Does it make sense to move the caveat regarding true negatives that is discussed in the table caption here?

File: metadata.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "title": "A Roadmap Toward Predicting Species Interaction Networks (Across Space and Time)",
```
Comment (gottacatchenall) [inline metadata.json:2]: i think in a zulip tim mentioned we should change the primer title (i think i part due to something in review 2), but worth asking

File: manuscript.md (hunk 1 above)
Comment (gottacatchenall) [reply manuscript.md:412]: yes

Comment (gottacatchenall) [reply manuscript.md:412]: also i think we need to change specificity/sensitivity to their rate synonyms, or at least add a parenthetical

File: metadata.json (hunk 2 above)
Comment (TanyaS08) [reply metadata.json:2]: indeed - also I call it the roadmap ms so I have no issues with the change!

**Comment by tpoisot** [comment]
STOP

WORKING

ON 

THIS

I 

AM

ON 

IT

:slightly_smiling_face:

**Comment by tpoisot** [comment]
@gottacatchenall the reviews are entirely f#ed up, please revert your changes (I left the merge conflicts because I need to run to a meeting), clean up, and make them again

**Comment by tpoisot** [comment]
@TanyaS08 / @gottacatchenall 
- [ ] final read through
- [ ] check that all tables and figures are referenced in text
- [ ] check that all references are rendering
- [ ] leave a `TKTP` text wherever I need to do something

I'll take some more time this afternoon to finish and send it to co-authors

**Comment by tpoisot** [comment]
@gottacatchenall can you also revert your edits on master? I'd like to have a clean file for latexdiff

**Comment by gottacatchenall** [comment]
reset master back to c271e3ca50d96fee96f73e39314a7b3b6937da9d

**Comment by gottacatchenall** [comment]
@TanyaS08 in order to do the above we need still need to decide on

- [ ] do we show uncertainty on AUC figures or only mention it in the text (re #128)
- [ ] do we add drawn on classifier labels on figure 1, or show both empirical and drawn AUC in figure 4
- [ ] do we do anything else to fig 1 visualization 
- [ ] update fig 1 caption 
- [ ] update fig 4 caption

**Comment by TanyaS08** [comment]
I think we drop fig 1 - transferring the real AUCs to figure for this overlaying real with hypothetical and then rather discuss uncertainty as opposed to visualising to keep fig 4 (will become fig 3) cleaner.

**Comment by TanyaS08** [comment]
This then sorts point 4 and I trust you on a caption for fig 4

**Comment by gottacatchenall** [comment]
- [ ] replace M and N with actual values (./manuscript.md line 35)
- [ ] tone check (./manuscript.md line 140)
- [ ] figure 1 caption rewrite
- [ ] there are citations in paragraph 2 not rendering 
- [ ] remove figure 4

**Comment by TanyaS08** [comment]
"potentially move comment about trait data into challenges section" I toyed with the idea but I felt like we spoke more about data availability rather than access - although we could still situate it there but would add a few more word potentially?

**Comment by gottacatchenall** [comment]
here's a final pitch for density of ROC/PR curves for random test/training splits to address #128 
prbootstrap
rocbootstrap

**Comment by tpoisot** [comment]
I am merging this PR into master to send to co-authors, but if you feel strongly about the bootstrap thing, let's work on another branch in the next 2 days or so

**Pull request merged**